semgrep-core/tests/Test.ml

```diff
@@ -258,6 +258,12 @@ let lang_regression_tests =
     let lang = Lang.Lua in
     regression_tests_for_lang files lang
   );
+  "semgrep Rust" >::: (
+    let dir = Filename.concat tests_path "rust" in
+    let files = Common2.glob (spf "%s/*.rs" dir) in
```

Comment:
Did you forget to add some test files in the PR?
The glob might raise an exn if there is no file in the directory I think.

Reply:
Yes, I did. I added them.